packages/sirius-web/backend/sirius-web-application/src/main/java/org/eclipse/sirius/web/application/views/explorer/services/ExpandAllTreePathProvider.java

```diff
@@ -76,7 +77,7 @@ public IPayload handle(IEditingContext editingContext, Tree tree, ExpandAllTreeP
         String treeItemId = input.treeItemId();
         Set<String> treeItemIdsToExpand = new HashSet<>();
         var object = this.getTreeItemObject(editingContext, treeItemId, tree);
-        if (object instanceof EObject) {
+        if (object instanceof EObject | object instanceof List<?>) {
```

Comment:
For the single `|` you are right, it is a mistake.
As far as the `List`  is concerned, getTreeItemObject in case of `Setting` returns the list of super types (cf ExplorerDescriptionProvider:313)
So I replace this line with : 
```java
 if (object instanceof EObject || treeItemId.startsWith(ExplorerDescriptionProvider.SETTING)) {
```

Reply:
Are you talking about this line in `ExplorerDescriptionProvider`:

```
                    result = internalObject.eSetting(internalObject.eClass().getEStructuralFeature(featureName));
```

Indeed the result is an instance of `List` but this `List` is the `Setting` object. It's not some random `ArrayList`. So unless I'm still mistaken, the test should not involve the `treeItemId` but it should test on `Setting`.